Extensions/Physics3DBehavior/JsExtension.js

```diff
@@ -289,6 +451,21 @@ module.exports = {
               'Define the weight of the object, according to its size. The bigger the density, the heavier the object.'
             )
           );
+        if (!behaviorContent.hasChild('massOverride')) {
+          behaviorContent.addChild('massOverride').setDoubleValue(0);
+        }
+        behaviorProperties
+          .getOrCreate('massOverride')
+          .setLabel(_('Mass override'))
```

Comment:
Should we display a unit?

Reply:
We could but we've never needed this unit before so we would have to add it.